src/pages/searchResults/[...selectedSubjects].js

```diff
@@ -1,22 +1,46 @@
 /* eslint-disable react/prop-types */
 import * as React from 'react'
-import { useState, useEffect } from 'react'
-import { Link, graphql } from 'gatsby'
-import { SubjectButton } from '../../components/buttons'
-import { StaticImage } from 'gatsby-plugin-image'
+import { graphql } from 'gatsby'
+import { SubjectTag, SkillsButton } from '../../components/buttons'
+
+const GetSelectedSubjectsArrayFromSearchQuery = ({ search }) =>
+  search
+    .substring(search.indexOf('=') + 1)
+    .replace(/\+/g, ' ')
+    .split(',')
+
+const GetSkillsFromSelectedSubjects = ({ selectedSubjects, skills }) =>
+  skills.filter(
+    ({ data: { Subjects } }) =>
+      Subjects != null && selectedSubjects.every((r) => Subjects.includes(r))
```

Comment:
what is r?

Reply:
also, i think for conventions sake it would be better to rename Subjects as lowercase subjects for use in the function, it's kinda confusing otherwise